src/software/thunderscope/common/common_widgets.py

```diff
@@ -329,3 +329,19 @@ def disable_radio_button(button_group):
         button.setChecked(False)
         button.clicked.disconnect()
         button.clicked.connect(lambda state, curr=button: curr.setChecked(False))
+
+
+def draw_robot(painter, rect, start_angle, span_angle):
+    """
+    Draws a robot bounded by the given rectangle with a chord defined by the given angles
+    :param painter:
+    :param rect:
+    :param start_angle:
+    :param span_angle:
+    :return:
```

Comment:
Could you add appropriate comments for the params? I'm having a hard time understanding what the angles due

Reply:
sorry forgot about this